mne_microstates.py

```diff
@@ -19,13 +19,14 @@
 from mne.utils import logger, verbose
 
 
-__version__ = '0.4dev0'
+__version__ = '0.5devkb0'
 
 
 @verbose
 def segment(data, n_states=4, n_inits=10, max_iter=1000, thresh=1e-6,
             normalize=False, min_peak_dist=2, max_n_peaks=10000,
-            return_polarity=False, random_state=None, verbose=None):
+            return_polarity=False, random_state=None, verbose=None,
+            return_best_gev=False, weight_array=None):
```

Comment:
```suggestion
            return_polarity=False, return_best_gev=False, random_state=None,
            verbose=None):
```
It's customary in MNE code to have the `random_state` and `verbose` parameters last. I think putting all the `return_*` parameters next to each other is more elegant. Yes, this would break backwards compatibility if someone is crazy enough to call this function without any parameters named (i.e. `segment(data, 4, 10, 1000, 1e-6, False, None, True)`), but who would be crazy enough to do that? ;)